changelogs/fragments/integration_test_fix.yml

```diff
@@ -1,3 +1,3 @@
 ---
 trivial:
-  - vyos_prefix_lists - fix vyos_prefix_lists in integration tests to resolve issues related to deletion.
+  - avoid_unsafe_conditions - Avoid unsafe conditions in integration tests to resolve issues related to assertion unsafe conditions.
```

Comment:
@brahmanim why are we updating an existing changelog entry?
Can we please add a new one?

Reply:
@KB-perByte I added a new avoid_unsafe_conditions.yml